src/styles/page.scss

```diff
@@ -0,0 +1,32 @@
+$color-white: #fff;
+$color-blue: #00acdc;
+$color-grin: #616070;
+$color-dark: #060b35;
```

Comment:
Create `_variables.scss` (use the underscore (_) prefix to create the no-compiled file), and move these variables there. This is an optional comment in this task.